lib/crystalball/source_diff/file_diff.rb

```diff
@@ -37,6 +37,18 @@ def new_relative_path
         git_diff.patch.match(/rename from.*\nrename to (.*)/)[1]
       end
 
+      def whitespace?
+        return false unless STRIPABLE_FILES.include?(Pathname(git_diff.path).extname)
+
+        patch = git_diff.patch
+
+        return true unless patch
+
+        patch.lines.select do |line|
+          line.start_with?('+', '-') && !line[1..-1].gsub(/\s/, '').empty?
+        end.empty?
```

Comment:
```ruby
patch.lines.none? do |line|
  ...
end
```

P.S. What about adding some spaces in begginning\end of line?

Reply:
it will work.